### PR TITLE
Correct docker command for starting Jaeger

### PR DESCRIPTION
**Pull request by SevInf**
Currently specified command is not enough to make it work.

### CODE REVIEW

File: content/200-concepts/100-components/02-prisma-client/250-opentelemetry-tracing.mdx [hunk 1]
```diff
@@ -163,7 +163,20 @@ The following screenshot shows an example trace visualization:
 To run Jaeger locally, use the following [Docker](https://www.docker.com/) command:
 
 ```console
-docker run --rm -d -p 13133:13133 -p 16686:16686 -p 4317:55680 jaegertracing/opentelemetry-all-in-one:latest
+docker run -d --name jaeger \                                                                                                      main
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/250-opentelemetry-tracing.mdx]:
```suggestion
docker run -d --name jaeger \
```

**Comment by janpio** [comment]
I think someone recently modified and simplified the command - we should double check how this could go so wrong.

**Review by andrew-walford-prisma — APPROVED** [review]
LGTM

**Comment by andrew-walford-prisma** [comment]
> I think someone recently modified and simplified the command - we should double check how this could go so wrong.

@janpio  

Do you mean someone modified the page in the docs? Looking at the page history, the line has already been the simplified form. Original commit where the page was added: https://github.com/prisma/docs/commit/effab66f11c0a2415cff54ad20ce09140bb272a2

**Comment by janpio** [comment]
I am probably remembering a change done to that PR while it was till open then - I am _pretty_ sure that I commented that the command does not work on windows being multi line like that.

It now again does not run on Windows of course:
```
C:\Users\Jan\Documents\throwaway>docker run -d --name jaeger \
docker: invalid reference format.
See 'docker run --help'.

C:\Users\Jan\Documents\throwaway>  -e COLLECTOR_ZIPKIN_HOST_PORT=:9411 \
'-e' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Jan\Documents\throwaway>  -e COLLECTOR_OTLP_ENABLED=true \
'-e' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Jan\Documents\throwaway>  -p 6831:6831/udp \
'-p' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Jan\Documents\throwaway>  -p 6832:6832/udp \
'-p' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Jan\Documents\throwaway>  -p 5778:5778 \
'-p' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Jan\Documents\throwaway>  -p 16686:16686 \
'-p' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Jan\Documents\throwaway>  -p 4317:4317 \
'-p' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Jan\Documents\throwaway>  -p 4318:4318 \
'-p' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Jan\Documents\throwaway>  -p 14250:14250 \
'-p' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Jan\Documents\throwaway>  -p 14268:14268 \
'-p' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Jan\Documents\throwaway>  -p 14269:14269 \
'-p' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Jan\Documents\throwaway>  -p 9411:9411 \
'-p' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\Jan\Documents\throwaway>  jaegertracing/all-in-one:latest
```

**Pull request merged**